src/app/query/stacks/tokens/non-fungible-tokens/non-fungible-token-metadata.query.ts

```diff
@@ -12,7 +12,7 @@ import { NftAssetResponse } from '../token-metadata.utils';
 import useGetNonFungibleTokenHoldingsQuery from './non-fungible-token-holdings.query';
 
 const queryOptions = {
-  refetchOnWindowFocus: true,
+  refetchOnWindowFocus: false,
```

Comment:
Good one, absolutely no reason to refetch this on focus. Reckon we could even put up the staletime